django/db/backends/base/features.py

```diff
@@ -157,8 +157,8 @@ class BaseDatabaseFeatures(object):
     # Support for the DISTINCT ON clause
     can_distinct_on_fields = False
 
-    # Can the backend use an Avg aggregate on DurationField?
-    can_avg_on_durationfield = True
+    # Can the backend use an aggregate on DurationField?
+    can_agg_on_durationfield = True
```

Comment:
I had imagined this flag would be removed, but if you think it will be useful for other backends, I don't mind keeping it. It's difficult to know which tests to apply it to without any backends in core that use it though.